fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -125,21 +124,20 @@ public Response capabilities() {
             Date startTime = new Date();
             checkInitComplete();
 
-            FHIRRequestContext ctx = FHIRRequestContext.get();
-            String tenantId = ctx.getTenantId();
-
             // Defaults to 60 minutes (or what's in the fhirConfig)
-            int cacheLength = FHIRConfigHelper.getIntProperty(PROPERTY_CAPABILITY_STATEMENT_CACHE, 60);
+            int cacheTimeout = FHIRConfigHelper.getIntProperty(PROPERTY_CAPABILITY_STATEMENT_CACHE, 60);
+            Configuration configuration = Configuration.of(Duration.of(cacheTimeout, ChronoUnit.MINUTES));
+
+            Map<Object, CapabilityStatement> cacheAsMap = CacheManager.getCacheAsMap(CAPABILITY_STATEMENT_CACHE_NAME, configuration);
+            CapabilityStatement capabilityStatement = cacheAsMap.computeIfAbsent(PRESENT, k -> computeCapabilityStatement());
```

Comment:
makes sense... what does the computeIfAbsent PRESENT part do... won't PRESENT always be there after the first call?